h/services/annotation_delete.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+from h.events import AnnotationEvent
+from h.models import Annotation
+
+
+class AnnotationDeleteService(object):
+    def __init__(self, request):
+        self.request = request
+
+    def delete(self, annotation):
+        """
+        Deletes the given annotation.
+
+        :param annotation: the annotation to be deleted
+        :type annotation: h.models.Annotation
```

Comment:
FYI you can write `:arg` instead of `:param`. Does exactly the same thing (it's an alias), just shorter. I've started doing that.